extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -39,6 +40,8 @@ export class AmpAdXOriginIframeHandler {
    * @param {!./amp-ad-3p-impl.AmpAd3PImpl|!../../amp-a4a/0.1/amp-a4a.AmpA4A} baseInstance
    */
   constructor(baseInstance) {
+    /** {!Window}*/
+    this.win = window;
```

Comment:
This is provide by the super class https://github.com/ampproject/amphtml/blob/master/src/base-element.js#L143

`window` may not be correct as AMP supports running its code in different iframes from the one it was first evaluated in.